docs/source/examples/tabular/checks/methodology/source/plot_unused_features.py

```diff
@@ -2,55 +2,66 @@
 """
 Unused Features
 ***************
-"""
+This notebook provides an overview for using and understanding the Unused Features check.
 
-#%%
+**Structure:**
 
-from deepchecks.tabular import Dataset
-import matplotlib.pyplot as plt
-from sklearn.ensemble import AdaBoostClassifier
-from sklearn.datasets import load_iris
-from deepchecks.tabular.checks import UnusedFeatures
-import pandas as pd
+* `How unused features affects my model? <#how-unused-features-affects-my-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
 
-#%%
-# Load data
-# =========
-# The dataset is the adult dataset which can be downloaded from the UCI machine learning repository.
-#
-# Dua, D. and Graff, C. (2019). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml].
-# Irvine, CA: University of California, School of Information and Computer Science.
+How unused features affects my model?
+=====================================
+Having too many features can lengthen training times and cause inaccuracies in the model, and is known as "The Curse
+of Dimensionality" or "Hughes Phenomenon". This is because the more dimensions there are, the larger the sample space,
+which can cause samples to be further apart from each other, as they are different in more dimensions, which badly
+affect distance measures on the samples. It might lead the model to overfit the data since it won't be able to
+generalize well when all the samples are far from each other.
```

Comment:
Contentwise need to make this more accurate -
The curse of dimensionality is not exactly that the "distances are large" but rather that the data samples are much more sparse, and cover less of the space (because the volume of the space grows exponentially as the dimensions grow), (thus it's harder to learn patterns without overfitting and requires much more samples).

Not important extra note - while the distances are large they're actually becoming more similar (the distance between 2 samples and 2 other samples is less different than in lower dimensionality, making clustering algorithms problematic).